prototype/addons/items/readme.md

```diff
@@ -0,0 +1,9 @@
+## Items Usage
+
+To create a new item:
+
+- 1- Create a new Resource and choose from [[EquipItem](/addons/items/classes/equip_item/equip_item.gd), [ConsumableItem](/addons/items/classes/consumable_item/consumable_item.gd), [ThrowableItem](/addons/items/classes/throwable_item/throwable_item.gd)]
+- 2- To Use the Items, Create a new ItemButton then add any created item to it's item property.
+- 3- To Create more Item types extend the Item class, look at the equip_item.gd for reference
+- 4- to add more textures check the Images class and add your image there then add it's path inside the Item's icon property
```

Comment:
4- to add more textures check the images folder and add your image there then add it's path inside the Item's icon property